apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -0,0 +1,762 @@
+  const assessmentSetAbbr = $('#usersTable').data('assessment-set-abbr');
+  const csrfToken = $('#usersTable').data('csrf-token');
+  const has_course_instance_permission_edit = $('#usersTable').data(
+    'has-course-instance-permission-edit',
+  );
+  const urlPrefix = $('#usersTable').data('url-prefix');
+  const bsTable = $('#usersTable').bootstrapTable({
+    buttons: {
+      studentsOnly: {
+        text: 'Students Only',
+        icon: 'fa-user-graduate',
+        attributes: { title: 'List only enrolled students' },
+        event: () => {
+          const table = $('#usersTable');
+          const filterOn = !table.data('filter-student-only');
+          table.data('filter-student-only', filterOn);
+
+          $('.columns button[name=studentsOnly]').toggleClass('active', filterOn);
+          if (assessmentGroupWork) {
+            table.bootstrapTable(
+              'filterBy',
+              { group_roles: 'Student' },
+              {
+                // the "filter" parameter has to be specified, hence we place a dummy placeholder here
+                filterAlgorithm: (row: { group_roles: string }, _: any) => {
+                  if (filterOn) {
+                    return row.group_roles.includes('Student');
+                  } else {
+                    return true;
+                  }
+                },
+              },
+            );
+          } else {
+            table.bootstrapTable('filterBy', filterOn ? { role: 'Student' } : {});
+          }
+        },
+      },
+    },
+    onPreBody() {
+      $('.spinning-wheel').show();
+    },
+    onResetView() {
+      $('.spinning-wheel').hide();
+
+      updateTotals($('#usersTable').bootstrapTable('getData'));
+
+      $('[data-toggle="popover"]').popover({
+        sanitize: false,
+        trigger: 'manual',
+        container: 'body',
+        html: true,
+        placement: 'auto',
+      });
+      $('.time-limit-edit-button')
+        .popover({
+          sanitize: false,
+          placement: 'right',
+          title() {
+            const row = $(this).data('row');
+            return row.action === 'set_time_limit_all'
+              ? 'Change Time Limits'
+              : row.open
+                ? 'Change Time Limit'
+                : 'Re-Open Instance';
+          },
+          container: 'body',
+          html: true,
+          trigger: 'click',
+          content: timeLimitEditPopoverContent,
+        })
+        .on('show.bs.popover', function () {
+          $($(this).data('bs.popover').getTipElement()).css('max-width', '350px');
+          $(this).find('.select-time-limit').change();
+        });
+    },
+    columns: tableColumns(assessmentGroupWork),
+  });
+  $(document).on('keydown', (event) => {
+    if (
+      (event.ctrlKey || event.metaKey) &&
+      String.fromCharCode(event.which).toLowerCase() === 'f'
+    ) {
+      $('.fixed-table-toolbar .search input').trigger('focus');
+      event.preventDefault();
+    }
+  });
```

Comment:
Added an extra conditional to handle the case of `ctrl + f` if the search input is already focused in [7efdf17](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/7efdf1794fea6c432e21d53c25ae86cac0585ebd). This will allow double-pressing `ctrl + f` to enable the default web browser functionality.